frontend/web/components/base/PermissionGate.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FC, ReactNode } from 'react'
```

Comment:
Can't we just adjust Permission.tsx to do all this ? I don't see why we need both. 

I think if we have both we'll start seeing some components use 1 vs the other, we run the risk of having too many similar components